### PR TITLE
Redo phonenumber with libphonenumber

### PR DESCRIPTION
**Pull request by ruthvik-17**
Use Google's Libphonenumber library's python version to detect phone numbers.

### CODE REVIEW

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

File: ner_v2/detectors/pattern/phone_number/phone_number_detection.py [hunk 1]
```diff
@@ -12,29 +13,25 @@ class PhoneDetector(BaseDetector):
...
             entity_name (str): A string by which the detected numbers would be replaced with
             on calling detect_entity()
             language (str, optional): language code of number text, defaults to 'en'
+            locale(str, optional): locale of the country from which you are dialing. Ex: 'en-IN'
         """
         self._supported_languages = NumberDetector.get_supported_languages()
-        super(PhoneDetector, self).__init__(language)
+        super(PhoneDetector, self).__init__(language, locale)
         self.language = language
-        self.entity_name = entity_name
```
Comment (amansrivastava17) [inline ner_v2/detectors/pattern/phone_number/phone_number_detection.py]: Dont remove `self.entity_name` and `self.tag`

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Pull request merged**